styles/base.css

```diff
@@ -16,11 +18,43 @@ li {
   border: 1px solid black;
 }
 
-nav > img:not(:last-of-type) {
+nav > a {
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+nav > a:not(:last-child) {
```

Comment:
What you wanna do with that statement?

Reply:
It's just for a border between the elements in the nav menu at the bottom